### PR TITLE
16 create components

### PR DESCRIPTION
**Pull request by snefrou2**
create all components for the agenda

### CODE REVIEW

**Review by magb7 — DISMISSED** [review]
See No Evil

File: src/components/AddNewHomework.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from 'react'
+import { View } from 'react-native'
+import styled from 'rn-css'
+
+const TextTitle = styled.Text`
+  font-size: 3em;
+`
+
```
Comment (LoicBrassart) [inline src/components/AddNewHomework.tsx:8]: Two consecutive whitespaces ? Shouldn't Prettier get rid of that ? :thinking:

File: src/components/Button/LeftArrow.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+const LeftArrowStyle = styled.Image`
+  width: 5em;
+  height: 3em;
+  pressduration: 0.3;
+`
+const LeftArrow = () => {
+  return (
+    <TouchableOpacity
+      onPress={() => {
+        console.log('fleche')
```
Comment (LoicBrassart) [inline src/components/Button/LeftArrow.tsx:13]: It makes me grunt to let this pass, even though I know why >_<

File: src/components/TaskDoneOverlay.tsx [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+  height: 707px;
+  background-color: rgba(0, 0, 0, 0.5);
+  border-radius: 3px;
+`
+
+const TaskDoneOverlay = () => {
+  return (
+    <Column>
+      <Image
+        style={{ width: '150px', height: '150px' }}
```
Comment (LoicBrassart) [inline src/components/TaskDoneOverlay.tsx]: Why use inline styles just there ?

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
- AddNewHomework is styled using `rn-css` while Checkbox uses `react-native`. Why use two distinct libraries for styling ?
- Where are my `;` ?
- ViewRightButton has no indentation rule
- DayColumn uses `rgb()` and `#` notations for colors: please choose one and stick to it for the whole project

File: src/components/Button/LeftArrow.tsx (hunk 2 above)
Comment (magb7) [reply src/components/Button/LeftArrow.tsx:13]: With Native, we can't write a onPress without something in the function :/

File: src/components/TaskDoneOverlay.tsx (hunk 3 above)
Comment (magb7) [reply src/components/TaskDoneOverlay.tsx]: Maybe because I was tired :)

File: src/components/AddNewHomework.tsx (hunk 1 above)
Comment (magb7) [reply src/components/AddNewHomework.tsx:8]: There aren't TWO consecutive whitespaces ! 
If you take a closer look, you can see that there is a backstick line 7 ! 

But I can understand you missed it, it's small.

**Review by magb7 — APPROVED** [review]
Everything's fine !  We add a README file with information on rn-css in it.

**Review by LoicBrassart — APPROVED** [review]
Saw no Evil

**Pull request merged**